ui/apps/platform/src/hooks/useFetchNetworkPolicies.ts

```diff
@@ -29,6 +29,8 @@ function useFetchNetworkPolicies(policyIds: string[]): Result {
                     setResult({ networkPolicies: [], error, isLoading: false });
                 });
         }
+
+        return () => setResult(defaultResultState);
```

Comment:
Oh, what's the benefit of this?

Reply:
it's useEffect clean up when the component unmounts